coralproject/talk
# Admin user details translations and wording

## What does this PR do?
Add translation keys on `UserDetails` and `ViewOptions` components to Admin. Also change E-mail to Identification and icon and show the provider to make it consistence when user register using Facebook.

## How do I test this PR?

- Setup your talk with `TALK_FACEBOOK_APP_ID` and `TALK_FACEBOOK_APP_SECRET`
- Create an `local` user and a `facebook` user
- Do some comments whit each user 
- Access Admin and click on the username
- See the translations and Identification field with new icon and provider (`local` or `facebook`)
- Check the Admin sidebar `View Options`
